src/site/content/en/blog/form-fatigue-guide-user-input/index.md

```diff
@@ -0,0 +1,273 @@
+---
+title: "Reduce form fatigue: guide users to proper input"
+author: mdiblasio
+date: 2019-05-04
+description: |
+  This post describes how to guide users to enter the proper information into forms.
+wf_blink_components: Blink>Accessibility
+tags:
+  - post
+  - accessibility
+  - ux
+---
+
+<!-- TODO (): update tags -->
+
+_This is post 2 of 5 of the [Reduce form fatigue series](../form-fatigue) that
+shows you how to build better forms for online stores._
+
+<!-- TODO (robdodson): is there a format we should use for this line that opens each article?  -->
+
+<!-- TODO (dutton): is there a format we should use for this line that opens each article?  -->
+
+<video autoplay loop muted playsinline>
+  <source src="video-typing-on-mobile-is-hard.mp4" type="video/mp4">
+</video>
+
+Typing on mobile is hard! In this guide you will learn how to reduce cart abandonment by guiding users
+through form-filling:
+
++   Use labels
++   Provide data format hints
++   Avoid invalid input
```

Comment:
The video and this line made me think that this post was going to cover validation but maybe that's yet to be added?

I'd suggest spinning that out into its own post and pairing the video with it instead.

Reply:
removed these bullets and moved video to validation post